### ISSUE TITLE
Pester v5

### ISSUE THREAD

**Issue by ThomasNieto** [PowerShellOrg/PSKoans#413]
This module is pinned to Pester v4. Are there plans to migrate to v5?

**Comment by vexx32** [comment]
Yep! We've already migrated to v5, but I hit a roadblock in testing things for v5 so I haven't been wanting to do a release until I have some way of testing all the functions in v5; specifically, the internal `Measure-Koan` function is utilizing Pester's discovery mechanic to get a more accurate accounting of koans... but you can't invoke it _inside_ a pester test, meaning we have no real way of verifying that it actually works correctly.

**Comment by HeyItsGilbert** [comment]
Resolved: the module now requires Pester >= 5.0.2 (`RequiredModules` in `PSKoans/PSKoans.psd1`), so the v4->v5 migration this issue asked about is complete. Closing.
